enterprise/frontend/src/embedding-sdk-bundle/components/public/dashboard/InteractiveDashboard/InteractiveDashboard.schema.ts

```diff
@@ -50,6 +50,8 @@ const propsSchema: Yup.SchemaOf<InteractiveDashboardProps> = Yup.object({
   withCardTitle: Yup.mixed().optional(),
   withDownloads: Yup.mixed().optional(),
   withTitle: Yup.mixed().optional(),
+  onVisualizationChange: Yup.mixed().optional(),
+  adHocQuestionHeight: Yup.mixed().optional(),
```

Comment:
We already have `drillThroughQuestionHeight` and `drillThroughQuestionProps.height` though. Could we apply those existing props to the ad-hoc question? Usually we use the term "drill-through" in place of ad-hoc in public-facing props.

cc @WiNloSt @npretto correct me if I'm wrong.